BUILDING.md

```diff
@@ -188,14 +188,19 @@ $ docker run --rm -v ${pwd}:/audacity/audacity/ -v ${pwd}/../build/linux-system:
 
 To find system packages, we rely on `pkg-config`. There are several packages that have broken `*.pc` or do not use `pkg-config` at all. For the docker image - we handle this issue by installing the correct [`pc` files](linux/build-environment/pkgconfig/).
 
+### Disabling AppImage support
+
+AppImage requires an additional executable called `findlib`. This executable will be installed to `bin`. If this behavior is not desired, it can be disabled 
```

Comment:
Does this default to `audacity_package_appimage=yes`? If so, why is it defaulting the build chain to assuming the appimage is the build instead of making the appimage build opt-in to this (which would require far fewer changes by downstream packagers)?

Reply:
It's one change for packagers :)

But I agree that `findlib` shouldn't get built by default, or perhaps it could be built by default but not installed to `bin`? My preference would be for it to get moved to `bin` in [package_appimage.cmake](https://github.com/audacity/audacity/blob/de2213ea2c794a294c1f242a75d5295512d9a73c/linux/package_appimage.cmake#L15-L17), which only runs if CPack is called, so no new CMake options would be required. This means somebody can compile Audacity and later decide to build the AppImage without having to reconfigure.